widgetssdk/src/main/java/com/glia/widgets/chat/adapter/UploadAttachmentAdapter.kt

```diff
@@ -180,7 +186,7 @@ class ViewHolder(
 
     private fun updateTitleAndStatusText(fileName: String, byteSize: String, status: Status) {
         val textColorRes =
-            if (status.isError) R.color.design_default_color_error else R.color.glia_base_normal_color
+            if (status.isError) com.google.android.material.R.color.design_default_color_error else R.color.glia_base_normal_color
```

Comment:
Import R with alias?

Reply:
Interesting suggestion 👍.
I thought the full path will show the origin of the `R` class, but we can find an alias that can describe it.